### PR TITLE
Smamindl/heatwave ml

### PR DESCRIPTION
**Pull request by smamindl**
Hi @jan-grzybek-ampere , can you review this PR? I have added the traditional models.

### CODE REVIEW

File: classic_ml/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+Each model has been trained on specific dataset and it should be tested using only that specific dataset. For ex Regression models are using diabetes dataset and classification models are using iris dataset.
+
+Dataset can be downloaded from here:
+```
+https://ampereaimodelzoo.s3.eu-central-1.amazonaws.com/sklearn2onnx/*.csv
+
+```
+
+### Metrics
```
Comment (jan-grzybek-ampere) [inline classic_ml/README.md:13]: What was the system under test? I guess Altra but how many cores etc?

File: classic_ml/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+Model description can be found here : https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html for all the models.
+
+Models can be downloaded from here:
+```
+ https://ampereaimodelzoo.s3.eu-central-1.amazonaws.com/sklearn2onnx/
+
+```
+Link for particular model is for ex: 
+```
+https://ampereaimodelzoo.s3.eu-central-1.amazonaws.com/sklearn2onnx/DecisionTreeRegressor_model.onnx
```
Comment (jan-grzybek-ampere) [inline classic_ml/README.md:53]: Could we list all models' urls + corresponding datasets?

File: classic_ml/LogisticRegressor/run.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+                        help="path to the model")
+    parser.add_argument("-p", "--precision",
+                        type=str, choices=["fp32"], required=True,
+                        help="precision of the model provided")
+    parser.add_argument("-b", "--batch_size",
+                        type=int, default=1,
+                        help="batch size to feed the model with")
+    parser.add_argument("-f", "--framework",
+                        type=str,
+                        choices=["pytorch", "ort"], required=True,
```
Comment (jan-grzybek-ampere) [inline classic_ml/LogisticRegressor/run.py:25]: pytorch is not available, ort could be made default

File: classic_ml/tabular_dataset.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    def __init__(self, file_path, batch_size=8, task='classification'):       
+        self.__batch_size = batch_size
+        self.__y_true = []
+        self.__y_pred = []
+        self.__k = 0
+        self.task = task
+        df = pd.read_csv(file_path)
+        self.X = df.iloc[:,:-1].values
+        self.y = df.iloc[:,-1].values
+        self.available_instances = float('inf')
```
Comment (jan-grzybek-ampere) [inline classic_ml/tabular_dataset.py:21]: is the number really infinite? this should be a finite value

**Review by jan-grzybek-ampere — APPROVED** [review]
Few suggestions in the comments but generally a good PR adhering to the general standard set by other tasks/networks 👍 thanks!

File: classic_ml/README.md (hunk 1 above)
Comment (smamindl) [reply classic_ml/README.md:13]: I used AltraMax.

File: classic_ml/README.md (hunk 2 above)
Comment (smamindl) [reply classic_ml/README.md:53]: sure.

File: classic_ml/tabular_dataset.py (hunk 4 above)
Comment (smamindl) [reply classic_ml/tabular_dataset.py:21]: Because we generate duplicate samples ideally we can have large num of samples.

**Pull request merged**